opnode/node/node.go

```diff
@@ -37,12 +37,11 @@ func (cfg *Config) Check() error {
 }
 
 type OpNode struct {
-	log          log.Logger
-	l1Source     eth.L1Source           // (combined) source to fetch data from
-	l1Downloader l1.Downloader          // actual downloader
-	l2Engines    []*driver.EngineDriver // engines to keep synced
-	ctx          context.Context        // Embeded CTX to be removed
-	done         chan struct{}
+	log       log.Logger
+	l1Source  l1.Source              // Source to fetch data from (also implements the Downloader interface)
+	l2Engines []*driver.EngineDriver // engines to keep synced
+	ctx       context.Context        // Embeded CTX to be removed
```

Comment:
typo in comment, and maybe we should just remove the ctx at this point.

Reply:
Fixed in next PR